src/CommandBinaryBuffer.ts

```diff
@@ -236,6 +289,60 @@ class CommandBinaryBuffer extends BinaryBuffer {
 
         [lowVoltageByte, lowAndHighVoltageByte, highVoltageByte].forEach(byte => this.setUint8(byte));
     }
+
+    getChannelsValuesWithHourDiff (): {hourAmount: number, channels: Array<IChannel>, date: Date} {
```

Comment:
```suggestion
    getChannelsValuesWithHourDiff (): ISomething {
```

?

Reply:
no idea, let's keep it like that for now